ui/components/MeshModelRegistry/MesheryTreeView.js

```diff
@@ -598,11 +599,11 @@ const MesheryTreeView = ({
           {treeComponent}
         </div>
       )}
-    </div>
+    </>
   );
 
   return (
-    <div style={{ width: '100%', height: '28.86rem' }}>
+    <div style={{ width: '100%', height: '100%', display: 'flex', flexDirection: 'column' }}>
```

Comment:
We can extract it into styled component

Reply:
Hi,
I have converted this div to styled component.
Please Check
Thanks